modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectFieldFormBase.tsx

```diff
@@ -107,9 +108,31 @@ export default function ObjectFieldFormBase({
 		return businessTypeMap;
 	}, [objectFieldTypes]);
 
+	const [defaultValue, setDefaltValue] = useState<ObjectState>();
+	const [defaultValueQuery, setDefaultValueQuery] = useState<string>('');
 	const [pickLists, setPickLists] = useState<PickList[]>([]);
 	const [pickListItems, setPickListItems] = useState<PickListItem[]>([]);
 
+	useEffect(() => {
+		const {objectFieldSettings} = values;
+
+		if (values.businessType === 'Picklist' && objectFieldSettings) {
+			const [objectFieldPicklist] = objectFieldSettings;
+			const {value} = objectFieldPicklist;
+			const {objectStates} = value as ObjectFieldPicklistSetting;
+			const defaultPicklistValue = objectStates.find(
+				(objectState) => objectState.key === values.defaultValue
```

Comment:
@M-IgorFranca, you can extract the key directly like:

```ts
const defaultPicklistValue = objectStates.find(
    ({key}) => key === defaultValue
)
```

Reply:
Done 👍🏾